apps/sms/views/shared/js/utils.js

```diff
@@ -473,6 +473,25 @@
       });
       return parsed;
     },
+    url(base, params) {
+      if (base.indexOf('?') === -1) {
```

Comment:
I didn't do it; I tried for some time and then thought I'd rather land this first and file a follow-up to investigate; with existing unit tests it should be easier.